src/browser/modules/Editor/language/consoleCommands.js

```diff
@@ -0,0 +1,123 @@
+export default [
+  {
+    name: ':help',
+    description: ' - learn about other topics',
+    commands: [
+      { name: 'commands' },
+      { name: 'play' },
+      { name: 'server' },
+      { name: 'cypher' },
+      { name: 'param' },
+      { name: 'params' },
+      { name: 'queries' },
+      { name: 'keys' },
+      { name: 'clear' },
+      { name: 'server-user' },
+      { name: 'match' },
+      { name: 'where' },
+      { name: 'return' },
+      { name: 'create' },
+      { name: 'merge' },
+      { name: 'delete' },
+      { name: 'detach-delete' },
+      { name: 'set' },
+      { name: 'foreach' },
+      { name: 'with' },
+      { name: 'load csv' },
+      { name: 'unwind' },
+      { name: 'start' },
+      { name: 'create-unique' },
+      { name: 'create-index-on' },
+      { name: 'starts-with' },
+      { name: 'ends-with' },
+      { name: 'contains' },
+      { name: 'rest' },
+      { name: 'rest-get'},
+      { name: 'rest-put'},
+      { name: 'rest-delete'},
+      { name: 'rest-post'}
+    ]
+  },
+  {
+    name: ':play',
+    description: ' - loads a mini-deck with either guide material or sample data',
+    commands: [
+      { name: 'intro' },
+      { name: 'graphs' },
+      { name: 'cypher' },
+      { name: 'movie graph' },
+      { name: 'northwind graph' },
+      { name: 'query template' }
+    ]
+  },
+  {
+    name: ':server',
+    description: ' - manage the connection to Neo4j',
+    commands: [
+      { name: 'status' },
+      { name: 'change-password' },
+      { name: 'connect' },
+      { name: 'disconnect' },
+      {
+        name: 'user',
+        commands: [
+          { name: 'add' },
+          { name: 'list' }
+        ]
+      }
+    ]
+  },
+  {
+    name: ':param',
+    description: ' - define a parameter'
+  },
+  {
+    name: ':params',
+    description: ' - show you a list of all your current parameters'
+  },
+  {
+    name: ':queries',
+    description: ' - list your servers and clusters running queries'
+  },
+  {
+    name: ':sysinfo',
+    description: ' - system information'
+  },
+  {
+    name: ':clear',
+    description: ' - remove all frames from the stream'
+  },
+  {
+    name: ':config'
+  },
+  {
+    name: ':schema - display indexes and constraints'
```

Comment:
` - display indexes and constraints` should be part of the description, not part of the name